<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DDNSSecretForm</class>
 <widget class="QWidget" name="DDNSSecretForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DDN密钥管理</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="pushButtonNewRecord">
     <property name="minimumSize">
      <size>
       <width>81</width>
       <height>21</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>21</height>
      </size>
     </property>
     <property name="text">
      <string>新增密钥(&amp;N)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>101</width>
         <height>16</height>
        </size>
       </property>
       <property name="text">
        <string>共 0 页，第 1 页</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButtonFirstPage">
         <property name="minimumSize">
          <size>
           <width>81</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>21</height>
          </size>
         </property>
         <property name="text">
          <string>首页(&amp;F)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonLastPage">
         <property name="minimumSize">
          <size>
           <width>81</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>21</height>
          </size>
         </property>
         <property name="text">
          <string>上一页(&amp;L)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonNextPage">
         <property name="minimumSize">
          <size>
           <width>81</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>21</height>
          </size>
         </property>
         <property name="text">
          <string>下一页(&amp;N)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonTailPage">
         <property name="minimumSize">
          <size>
           <width>81</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>21</height>
          </size>
         </property>
         <property name="text">
          <string>尾页(&amp;T)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QSpinBox" name="spinBoxPage"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonGoPage">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>21</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>21</height>
          </size>
         </property>
         <property name="text">
          <string>跳转(&amp;G)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
